config/settings.yml

```diff
@@ -38,6 +38,14 @@ user:
     - sulair:proxy-access
     - lane:proxy-access
   locations:
+    # Codes from https://github.com/sul-dlss/cocina-models/blob/8fc7b5b9b0e3592a5c81f4c0e4ebff5c926669c6/openapi.yml#L1330-L1339
+    # labels from https://consul.stanford.edu/display/chimera/Rights+Metadata+Locations
+    labels: { spec: 'Special Collections reading room',
```

Comment:
Typically we don't put curly braces in yaml

Reply:
Fixed